graphics/ctpv/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github   1.0
+PortGroup           makefile 1.0
+
+github.setup        NikitaIvanovV ctpv 6f364cf383e87d2ab51bee307bbd149c2fb252db
+github.tarball_from archive
+
+version             1.0
+categories          graphics
+license             MIT
+platforms           darwin
+maintainers         {@suhailskhan} \
+                    openmaintainer
+description         Image previews for lf file manager
+long_description    ctpv is a previewer utility for terminals. It supports previews for \
+                    source code, archives, PDF files, images and videos. \
+homepage            https://nikitaivanovv.github.io/ctpv
+
+checksums           rmd160  32236eb8b1b6933bdc1fbe93e76c132e316c87e8 \
+                    sha256  21b64c019f1e6332fb6500e488d4f4914eb912c29fba09c084a8f82bf8ddb9fa \
+                    size    2010486
+
+depends_lib-append  lib:libcrypto:openssl \
+                    lib:libmagic:libmagic               
```

Comment:
what other port could provide `libmagic` library besides the `libmagic` port; it there is none, then this should just use the typical `port:libmagic` dependency declaration.